### DISCUSSION TITLE
What is “EQ Embed” quality?

### DISCUSSION THREAD

**Discussion by Bucchiac** [Goldenfreddy0703/Otaku#311] [General]
I usually go with 1080p, but that’s only available from H!Anime which has been so problematic lately.

Some providers like Gogo have EQEmbed but I’m confused as to what that stands for and what the resolution is. Anyone know?

**Comment by Goldenfreddy0703** [comment]
EQ basically stands for Embed Quality where sometimes if we don't know what the quality of an embed is, we have it show EQ instead.
